runtime.c

```diff
@@ -385,6 +392,38 @@ main (int argc, char *argv[])
         exit(0);
     }
 
+    if (arg && strcmp(arg,"appimage-portable-home")==0) {
+        char portable_home[2048];
+        int mkdir_ret;
+
+        sprintf(portable_home, "%s.home", appimage_path);
+        mkdir_ret = mkdir(portable_home, S_IRWXU);
+        if (mkdir_ret == 0)
+            printf("Portable home folder created at %s\n", portable_home);
```

Comment:
Let's call it "directory" rather than "folder", as that is the standard terminology on Linux. Also in the other lines below.